### ISSUE TITLE
Standalone version of WebRTCCamera outside of HASS?

### ISSUE THREAD

**Issue by cerahmed** [AlexxIT/WebRTC#568]
Hi again Alex,

I've been using this plugin for a while in my Home Assitant and I'm fond of the functionality of the card. Specifically, the minimal contorls (fullscreen, mute, etc.), and the DigitalPTZ functionality.

As I've mentioned in my recent go2rtc repo, I'm currently integrating go2rtc to my django project, in which I'd like to have the same functionality of the card on my own html.

Is there a standalone version of the WebRTCCamera class that does not depent on HASS?

**Comment by AlexxIT** [comment]
No. This card depend on Hass settings and Hass API. But you can try to modify it
